scripts/build_test.py

```diff
@@ -903,20 +903,39 @@ def _mock_safe_delete_file(unused_filepath):
         build.safe_delete_file(
             'core/tests/data/third_party/js/third_party.min.js.map')
 
+    def test_clean(self):
+        check_function_calls = {
+            'safe_delete_directory_tree_gets_called': 0,
+        }
+        expected_check_function_calls = {
+            'safe_delete_directory_tree_gets_called': 3,
+        }
+
+        def mock_safe_delete_directory_tree(unused_path):
+            check_function_calls['safe_delete_directory_tree_gets_called'] += 1
+
+        with self.swap(
+            build, 'safe_delete_directory_tree',
+            mock_safe_delete_directory_tree):
```

Comment:
```suggestion
        with self.swap(
            build, 'safe_delete_directory_tree',
            mock_safe_delete_directory_tree
        ):
```

ditto elsewhere (it is bit weird that the indentation is the same for code in with and in the with-block).

Reply:
Done.